apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html

```diff
@@ -8,12 +8,12 @@
   <div class="align-items-center d-flex flex-column">
     <mat-form-field appearance="outline" class="w-100">
       <mat-label i18n>Security Token</mat-label>
-      <textarea
+      <input
         cdkTextareaAutosize
```

Comment:
Is this still working? If not, please remove.

Reply:
Its not working with input field. I removed it